drydock/templates/drydock/k8s/ingress/lms.yml

```diff
@@ -0,0 +1,116 @@
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: ingress-lms
+  namespace: {{ K8S_NAMESPACE }}
+  {%- if DRYDOCK_AUTO_TLS and not DRYDOCK_CUSTOM_CERTS %}
+  annotations:
+    cert-manager.io/issuer: letsencrypt
+  {%- endif %}
+  {%- if DRYDOCK_ENABLE_SCORM and DRYDOCK_BYPASS_CADDY %}
```

Comment:
There's a problem here. When the `if` statement just above evaluates to `false`, the `annotations` key is not written in the YAML, which could lead to a problem with this SCORM annotation. I would extract the `annotations` key in the if above and would check if it works

Reply:
Remember Scorm annotation should be applied to [CMS](https://github.com/eduNEXT/drydock/blob/main/drydock/patches/caddyfile-cms) as well

Reply:
Addressed, it appears to work fine cause `commonAnnotations` from `Kustomization` add some annotations